tests/utils/tools.php

```diff
@@ -630,31 +633,31 @@ function configureFailPoint(MongoDB\Driver\Manager $manager, $failPoint, $mode,
         $doc["data"] = $data;
     }
 
-    $cmd = new MongoDB\Driver\Command($doc);
-    $result = $manager->executeCommand("admin", $cmd);
-    $arr = current($result->toArray());
-    if (empty($arr->ok)) {
-        var_dump($result);
-        throw new RuntimeException("Failpoint failed");
+    try {
+        $cmd = new MongoDB\Driver\Command($doc);
+        $result = $manager->executeCommand("admin", $cmd);
+    } catch (Exception $e) {
+        /* don't care if the fail point can't be set, it's MongoD version
+         * dependent */
     }
-    return true;
 }
 
 function failMaxTimeMS(MongoDB\Driver\Manager $manager) {
     return configureFailPoint($manager, "maxTimeAlwaysTimeOut", array("times" => 1));
 }
 
-function getMOUri() {
-    if (!($HOST = getenv("MONGODB_ORCHESTRATION_HOST"))) {
-        $HOST = "192.168.112.10";
+function getMongoOrchestrationURI()
```

Comment:
I reckon we can do away with this method entirely and simply define the appropriate constants in `basic.inc` as we do for others. Consider:

```php
define('MONGODB_ORCHESTRATION_HOST', getenv('MONGODB_ORCHESTRATION_HOST') ?: 'localhost');
define('MONGODB_ORCHESTRATION_PORT', getenv('MONGODB_ORCHESTRATION_HOST') ?: '8889');
define('MONGODB_ORCHESTRATION_URI', sprintf('http://%s:%s/v1', MONGODB_ORCHESTRATION_HOST, MONGODB_ORCHESTRATION_PORT));
```

If the individual host and port values aren't needed as their own constant, you could also use local variables with the same `?:` expression purely to ensure `MONGODB_ORCHESTRATION_URI` gets set properly.